package day10.exception.throw_;

public class ThrowEx01 {

	public static void main(String[] args) {
		
		/*
		 * throw - 예외 강제로 발생시키기
		 * 
		 * 메서드나 생성자에서 throw new 예외클래스()
		 * 를 만나면 예외가 생성됩니다.
		 * 
		 */
		
		int result = calc(-10);
		System.out.println(result);
		
		
	}
	
	public static int calc(int n) {
		
		int sum = 0;
		for(int i = 1; i <= n; i++) {
			sum += i;
		}
		
		return sum;
	}
	
	
	
	
	
	
	
	
}
